src/Tickets/Commerce/Gateways/Stripe/Client.php

```diff
@@ -142,7 +200,51 @@ static function ( $item ) {
 
 		$payment_intent = $this->post( $url, $query_args, $args );
 
-		$this->store_payment_intent( $payment_intent );
+		if ( ! isset( $payment_intent['id'] ) && ! empty( $payment_intent['errors'] ) ) {
+
+			if ( $this->count_retries() ) {
+				$this->delete_payment_intent_transient();
+				$this->start_fallback_mode();
+				$this->log_stripe_errors( $payment_intent );
+
+				return $this->create_payment_intent();
+			}
+
+			$this->log_stripe_errors( $payment_intent, false );
+
+			$payment_intent['errors'][0] = [
+				'et_could_not_create_stripe_order',
+				__( 'There was an error enabling Stripe on your cart. More information is available in the Event Tickets settings dashboard. Please contact the site administrator for support.', 'event-tickets' ),
+			];
+
+			return $this->store_payment_intent( $payment_intent );
+		}
+
+		$this->stop_fallback_mode();
+
+		return $this->store_payment_intent( $payment_intent );
+	}
+
+	/**
+	 * Store Payment Intent creation errors for admins to resolve
+	 *
+	 * @since TBD
+	 *
+	 * @param array $payload returned payload from failed attempts to create a PaymentIntent object
+	 */
+	private function log_stripe_errors( $payload, $recoverable = true ) {
+		$payload = array_shift( $payload );
+
+		foreach ( $payload as $logs ) {
+			$error['type']        = $logs[0];
+			$error['message']     = $logs[1];
+			$error['time']        = gmdate( 'c', time() );
```

Comment:
Use `Date_Utils::build_date_object` object, save the date with the format `Date_Utils::DBDATETIMEFORMAT`